components/clp-package-utils/clp_package_utils/general.py

```diff
@@ -309,6 +306,19 @@ def generate_container_config(
     return container_clp_config, docker_mounts
 
 
+def generate_docker_compose_container_config(clp_config: CLPConfig) -> CLPConfig:
```

Comment:
`generate_container_config()` should eventually be deprecated when we migrate the other (compress / decompress / search / management) scripts to docker compose